pkgs/shells/zsh/zsh-autopair/default.nix

```diff
@@ -2,17 +2,15 @@
 
 stdenv.mkDerivation rec {
   pname = "zsh-autopair";
-  version = "1.0";
+  version = "2024-07-14";
 
   src = fetchFromGitHub {
     owner = "hlissner";
     repo = "zsh-autopair";
-    rev = "v${version}";
-    sha256 = "1h0vm2dgrmb8i2pvsgis3lshc5b0ad846836m62y8h3rdb3zmpy1";
+    rev = "449a7c3d095bc8f3d78cf37b9549f8bb4c383f3d";
+    hash = "sha256-3zvOgIi+q7+sTXrT+r/4v98qjeiEL4Wh64rxBYnwJvQ=";
   };
 
-  strictDeps = true;
```

Comment:
why remove this?

Reply:
Since there's no `buildInputs` nor `nativeBuildInputs`, I thought it wouldn't matter much in this case. Just a cleanup ^^.